src/envoy/core.clj

```diff
@@ -120,9 +121,11 @@
         (if-let [type-key (:type definition)]
           (types/parse type-key v)
           v))
-      ; Check if the var has missing behavior.
-      (behave! :missing-access (:missing definition)
-               "Access to env variable %s which has no value" k))
+      (if-let [default (:default definition)]
+        default
```

Comment:
I would like to see logging about this being a default value, and log a conflict if `:missing` and `:default` are both defined.